README.md

```diff
@@ -52,47 +52,62 @@ npm install @opentelemetry/api
 The Dynatrace exporter is added and set-up like this:
 
 ```js
-const { configureDynatraceMetricExport } = require('@dynatrace/opentelemetry-exporter-metrics');
+'use strict';
+
 const { diag, DiagConsoleLogger, DiagLogLevel } = require('@opentelemetry/api');
 const { Resource } = require('@opentelemetry/resources');
 const { MeterProvider } = require('@opentelemetry/sdk-metrics');
+const { configureDynatraceMetricExport } = require('@dynatrace/opentelemetry-exporter-metrics');
+
 
-// optional: set up logging for OpenTelemetry
-diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.ALL);
+diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.ALL)
+
+// Configure a MeterProvider
+const provider = new MeterProvider({
+  resource: new Resource({
+    'service.name': 'opentelemetry-metrics-sample-dynatrace'
+  })
+});
 
-// configure the export to Dynatrace
 const reader = configureDynatraceMetricExport(
   // exporter configuration
   {
-    prefix: 'my_prefix', // optional
-    defaultDimensions: [   // optional
```

Comment:
Since this is the readme it might be a good idea to retain the optional configs as an example of their use

Reply:
Done in [0cee3c9](https://github.com/dynatrace-oss/opentelemetry-metric-js/pull/37/commits/0cee3c9b92bf3df3f495dc3299059305ada0842b) :slightly_smiling_face: